src/Microsoft.AspNet.Razor.Runtime/TagHelpers/TagHelperExecutionContext.cs

```diff
@@ -134,6 +134,26 @@ public void Add([NotNull] ITagHelper tagHelper)
         }
 
         /// <summary>
+        /// Tracks the minimized HTML attribute in <see cref="AllAttributes"/> and <see cref="HTMLAttributes"/>.
+        /// </summary>
+        /// <param name="name">The minimized HTML attribute name.</param>
+        public void AddHtmlAttribute([NotNull] string name)
```

Comment:
I can't leave out the helper because it needs to do specific work to add a minimized attribute. As for renaming: I don't feel like it warrants an entirely new method name for a different form of an HTML attribute, that to me makes less sense. Going to leave as is.